### PR TITLE
LPS-113561 Avoid use of Liferay.provide in portal-settings-authentication-ldap-web

### PR DESCRIPTION
**Pull request by julien**
As part of [LPS-113561](https://issues.liferay.com/browse/LPS-113561) we eventually want to deprecate the `Liferay.provide` function.

To test this change:

- From the "Global Menu" select the "Control Panel" tab
- Click on "Configuration > Instance Settings"
- Scroll down the the "Security" section and click on "LDAP"
- Click on the "Servers" tab
- Click on the "Add" button
- Enter a name for your server
- Under "Default Values" select one of the options
- Scroll down and click on the "Test LDAP Connection" button

As a result a modal window should open, depending on the server configuration you entered it should either tell you that the connection to the LDAP server was successful or that it failed, and no JS errors should appear in your browser's console

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by wincent** [comment]
Same as [here](https://github.com/wincent/liferay-portal/pull/343#issuecomment-642511844), going to see if we can just rewrite to avoid AUI dependencies here entirely.

**Comment by julien** [comment]
Here's a screenshot of how it looked before:

ldap-before

And how it looks after:

ldap-after

File: modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/resources/META-INF/resources/com.liferay.portal.settings.web/edit_ldap_server.jsp [hunk 1]
```diff
@@ -502,140 +502,151 @@ renderResponse.setTitle((ldapServerId == 0) ? LanguageUtil.get(resourceBundle, "
...
+</script>
 
-	Liferay.provide(
-		window,
-		'<portlet:namespace />testSettings',
-		function (type) {
-			var A = AUI();
-
-			var url = null;
+<aui:script>
```
Comment (wincent) [inline modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/resources/META-INF/resources/com.liferay.portal.settings.web/edit_ldap_server.jsp:507]: May as well unify these two consecutive script tags. I'll push a commit for that.

**Comment by wincent** [comment]
Given your test plan and screenshots, going to trying forwarding and trust CI to do its job. 😂

**Comment by wincent** [comment]
ci:forward

File: modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/resources/META-INF/resources/com.liferay.portal.settings.web/edit_ldap_server.jsp (hunk 1 above)
Comment (julien) [reply modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/resources/META-INF/resources/com.liferay.portal.settings.web/edit_ldap_server.jsp]: Thanks!

**Pull request closed without merging**